client/src/connection/rest/util.ts

```diff
@@ -38,12 +38,18 @@ export const getResourceIdFromItem = (item: ContentItem): string | null => {
   return getLink(item.links, "GET", "self")?.uri || null;
 };
 
-export const getSasContentUri = (item: ContentItem, readOnly?: boolean): Uri =>
+export const getSasContentUri = (
```

Comment:
Instead of having a nested ternary here, could we just _always_ pass in `fullPath`. So the function would become something like
```
getSasContentUri(item: ContentItem, path: string, readOnly?: boolean)
```
and we'd just _always_ use path

Reply:
I feel like an earlier version of your changes was using a nested ternary, which seemed unnecessary and made the code a bit harder to follow. I like how you cleaned this up.